src/styles/utils/animations.scss

```
@@ -0,0 +1,19 @@
+@keyframes rotate-minute {
+  from {
+    transform: translate(-50%, 0) rotate(0);
+  }
+
+  to {
+    transform: translate(-50%, 0) rotate(360deg);
+  }
+}
+
+@keyframes rotate-second {
+  from {
+    transform: translate(-50%, -47.5%) rotate(0);
+  }
+
+  to {
+    transform: translate(-50%, -47.5%) rotate(360deg);
+  }
+}
```

The rotation script is the same, so you only need to use one
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0);
  }

  to {
    transform: rotate(360deg);
  }
}
```

---

> The rotation script is the same, so you only need to use one

Since we remove 'translate' property with the new keyframe, all the positioning becomes wrong, that's why I continue using "translate" property along with "rotate".